applications/Chat/coati/experience_maker/naive.py

```diff
@@ -24,7 +24,7 @@ def make_experience(self, input_ids: torch.Tensor, **generate_kwargs) -> Experie
         action_log_probs = self.actor(sequences, num_actions, attention_mask)
         base_action_log_probs = self.initial_model(sequences, num_actions, attention_mask)
         value = self.critic(sequences, action_mask, attention_mask)
-        r = self.reward_model(sequences, attention_mask)
+        r = self.reward_model(sequences, attention_mask, action_mask=action_mask)
```

Comment:
To compute reward，we should use the whole sequence as training rm

Reply:
Yes you are right, here we should follow the rm training. 

This remind me of another issue: either log-sig or log-exp computes (chosen_reward - reject_reward) which actually remove the prompt's reward if we ignore the average computation.  Does this means the prompt part of the sequence are under Inadequate training?